.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  workflow_call:
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]

```

Comment:
I'd like to fetch the node version from the `.nvm` file so we can update that in one place only. The setup node action has built-in support for doing that, but I'm not sure how to do that while still using the matrix.

https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

```
- uses: actions/setup-node@v3
  with:
    node-version-file: '.nvmrc'
```

Reply:
Can we configure it to use a default if the `.nvmrc` file isn't present?

Reply:
I can't find an example that does this, but maybe it'll work if you specify multiple.

```yaml
- uses: actions/setup-node@v3
  with:
    node-version-file: '.nvmrc'
    node-version: 16
```

Reply:
![image](https://github.com/microsoft/vscode-azuretools/assets/36966225/dbf28d8e-9bce-4229-8e40-e77a6831e830)

Reply:
Looks like both ADO and GH Actions equally support using `.nvmrc`, so I'll change the tasks to use that. And the readme to reflect that that's the requirement, instead of Node 16.